File: python/selection-sort.py
def main():
    arr = [2,56,4,2,345,58,5,346,25,1,27,37,16,38,48,57,62,17]

    #recorro el array
    #guardo el primer numero
    #voy comparando cada elemento con el numero guardado
    #si es menor lo cambio al indice actual

    for i in range(0, len(arr)):
        minor = i
        for j in range(i+1, len(arr)):
            if arr[minor] > arr[j]:
                minor = j
        temp = arr[i]
        arr[i] = arr[minor]
        arr[minor] = temp
        print(arr)

if __name__ == "__main__":
    main()

File: js/selection-sort.js
let arr = [1,4,65,8,4,2,26,48,50,15,45,67,19,38,58,52,473,79,24,13,24,46,63,67];

for(let i = 0; i < arr.length; i++) {
  let minor = i;
  for(let j = i+1; j < arr.length; j++) {
    if (arr[j] < arr[minor]) {
      minor = j;
    }
    let temp = arr[i];
    arr[i] = arr[minor];
    arr[minor] = temp;
  }
  console.log(arr);
}
